src/Core/Asset/GeometryData.hpp

```diff
@@ -73,101 +73,147 @@ class RA_CORE_API GeometryData : public AssetData
     inline std::size_t getVerticesSize() const;
 
     /// Return the list of vertices.
-    inline Vector3Array& getVertices();
+    [[deprecated( "Use verticesWithLock() "
+                  "instead." )]] inline Vector3Array&
+    getVertices();
 
-    /// Return the list of vertices.
-    inline const Vector3Array& getVertices() const;
+    /// Return the (const) list of vertices.
+    [[deprecated( "Use vertices() instead. " )]] inline const Vector3Array& getVertices() const;
 
     /// Set the mesh vertices.
-    /// \note In-place setting with getVertices() is preferred.
+    /// \note Use getAttrib( const Geometry::MeshAttrib& name ) instead.
     template <typename Container>
-    inline void setVertices( const Container& vertexList );
+    [[deprecated( "Use getAttrib( const Geometry::MeshAttrib& name ) instead. " )]] inline void
+    setVertices( const Container& vertexList );
 
     /// Return the list of lines.
     /// \note For line meshes only.
-    inline Vector2uArray& getEdges();
+    [[deprecated(
+        "Use findIndexDataWithLock( const std::string& name, const bool& firstOccurrence ) "
+        "instead." )]] inline Vector2uArray&
```

Comment:
```suggestion
    [[deprecated(
        "Use findIndexDataWithLock<Vector2ui>( \"in_edge\" ) instead." )]] inline Vector2uArray&
```
Depreciation message should give the right function to use, instead of the generic one (apply to all deprecated methods).

Reply:
Yes, I will change that